docs/docs/workflows/scale.md

```diff
@@ -88,6 +97,15 @@ To increase the number of control-plane nodes, follow these steps:
 2. **Edit** the **control-plane** instance group.
 3. Set the new (increased) **number of instances** and **save**.
 
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+:::caution
+
+Scaling isn't yet implemented for AWS. If you require this feature, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md)!
```

Comment:
Why do we want users to create issues for everything we likely should support in the first place (and likely will add later anyways)? Or do we plan on abandoning AWS after the release of 2.2.0 until we get user feedback?

Reply:
Ah, bad wording on my side, but I see.

I wasn't so sure about whether it was just not available since AWS was just recently implemented and therefore not everything is ready yet or if we actually deprioritize it / don't want to work on it for now as technically AWS also offers the weakest security model of the CSPs we support. 

An alternative suggestion I have is to reword this as _will become available in a later release_ but still suggest opening an issue if this is a priority for someone. I think that is more neutral and reflects that AWS is more in an early state.

But don't need to change it then if we actually don't want to focus on AWS for now until we know users actually want to use it. I think that's fine then.

Reply:
I hinted at the fact that we have those tasks in our backlog: "Scaling isn't **yet** implemented for AWS.", users should engage with us (via issues) if this is an urgent need.

I will resolve this conversation for now, feel free to reopen if you feel we should change/improve the wording.